champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/data/DaggerDataManager.java

```diff
@@ -0,0 +1,43 @@
+package me.mykindos.betterpvp.champions.champions.skills.skills.ranger.data;
+
+import lombok.Getter;
+import org.bukkit.entity.Player;
+import java.util.Map;
+import com.google.inject.Singleton;
+
+import java.util.HashMap;
+import java.util.Set;
+
+@Singleton
+public class DaggerDataManager {
+
+    @Getter
+    private final Map<Player, DaggerData> daggerDataMap = new HashMap<>();
+
+    @Getter
+    private static final DaggerDataManager instance = new DaggerDataManager();
```

Comment:
If this is a `@Singleton`, you can just inject this class through a constructor, avoid having `static final` members.